### ISSUE TITLE
Package.json main entry missing.

### ISSUE THREAD

**Issue by RichardMartinez** [jzaefferer/jquery-validation#1846]
### Subject of the issue
It appears that the packack.json [main entry](https://github.com/jzaefferer/jquery-validation/blob/master/package.json#L35) should be updated. It looks like the structure existed in a previous release: [15.1](https://github.com/jzaefferer/jquery-validation/blob/1.15.1/dist/jquery.validate.js), but not [currently](https://github.com/jzaefferer/jquery-validation/dist/jquery.validate.js).

### Your environment
Any.

### Expected behaviour
The main entry points to the main file that should be loaded, when the package is required/imported.

### Actual behaviour
Any require/import fails, when using package name.

**Comment by staabm** [comment]
the `dist` folder is created in the release-process and is not part of the actual repository. we generate it automatically with our build script.

what the underlying problem you are trying to solve?

Will close for now. Feel free to ask further questions regarding this topic here. We can re-open if required.

**Comment by RichardMartinez** [comment]
If I did the following I would have a problem, because the main key/value doesn't exist until it is built:

`npm i jquery-validation --save`

and then in a main.js, for instance, I would 

`import "jquery-validation";` or `require('jquery-validation');`

Is this node package now intended for someone that plans on contributing, because I was able to do this with a previous version?

Maybe I am missing something. 

Also... If I wanted to use a relative path, would this be the file that I point to: https://github.com/jzaefferer/jquery-validation/blob/master/src/core.js

Thanks.
